### ISSUE TITLE
fix flacky screen-reader tests

### ISSUE THREAD

**Issue by mfranzke** [db-ux-design-system/core-web#6795]
It's probably an issue because of the runner in the background, a retry on the same runner results in the same output. We should look at the tests and see if we can add some workaround for the failing tests. We should also enable the https://playwright.dev/docs/videos to see if the tests just fails, because of the timing or something else.

_Originally posted by @nmerget in https://github.com/db-ux-design-system/core-web/issues/6724#issuecomment-4540789750_
